agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/ConnectionString.java

```diff
@@ -39,9 +39,19 @@
 
   private ConnectionString() {}
 
-  public static void parseInto(String connectionString, TelemetryClient targetConfig)
+  public static void parseInto(String connectionString, TelemetryClient telemetryClient)
       throws InvalidConnectionStringException {
-    mapToConnectionConfiguration(getKeyValuePairs(connectionString), targetConfig);
+    if (Strings.isNullOrEmpty(connectionString)) {
+      telemetryClient.setInstrumentationKey(connectionString);
+
+      // reset endpoint to default for ingestion, live metric and profiler
+      EndpointProvider endpointProvider = telemetryClient.getEndpointProvider();
+      endpointProvider.setIngestionEndpoint(endpointProvider.getIngestionEndpoint());
```

Comment:
this looks like it's just setting the same value back?

Reply:
yup, please check the update. i pushed too quick.